### PR TITLE
Clarify PR reviewing and facilitating

### PR DESCRIPTION
**Pull request by tigrannajaryan**
We recently introduced the automatic assignments of PRs to reviewers
and to facilitators. This change explains the process.

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
@jrcamp @anuraaga @jpkrohling @james-bebbington @owais @nilebox @dmitryax please read the proposed changes.

**Review by bogdandrutu — APPROVED** [review]

File: README.md
Comment (jpkrohling) [inline README.md:96]: Not part of this PR, but I noticed that the approvers group is also added as a reviewer to PRs. Is it possible to disable that?

**Review by jpkrohling — APPROVED** [review]

File: README.md
Comment (bogdandrutu) [reply README.md:96]: It should not be, we need one approval from that group as well.

Comment (bogdandrutu) [reply README.md:96]: The approval from the components owner does not count as approval of the PR unfortunately unless we open permission to everyone to have write permission, which opens possibility to do releases etc.

Comment (jpkrohling) [reply README.md:96]: I had the impression that I started receiving notifications even for PRs that weren't for me to review, but I might be wrong. I'll leave a comment here if this happens.

File: README.md
Comment (anuraaga) [inline README.md:102]: Dunno if it helps but just in case, I always have my eye on https://github.com/notifications (thanks @jkwatson!). If the assigned tab has anything I prioritize it over everything else.

**Review by anuraaga — APPROVED** [review]

File: README.md
Comment (bogdandrutu) [inline README.md:100]: The facilitator I think needs to do a review as well after the component owner reviewed it, if the component owner is not one of the "approvers". Does that make sense? Otherwise one of us (9) will need to do an official review anyway

Comment (tigrannajaryan) [reply README.md]: Edited, PTAL.

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**
